Platform.PCL/Realm.Sync.PCL/RealmExtensionsPCL.cs

```diff
@@ -18,8 +18,16 @@
 
 namespace Realms.Sync
 {
+    /// <summary>
+    /// A set of extension methods that provide Sync-related functionality on top of Realm classes.
```

Comment:
They will still show up in the API reference docs. Which doesn't invalidate the `[Browsable(false)]` suggestion, just means that we still have to document them.